src/main/java/com/iggroup/universityworkshopmw/domain/services/PriceGeneratorService.java

```diff
@@ -40,9 +39,8 @@ private void updateMarketPrices() {
                .currentPrice(newMarketPrice)
                .build()
          );
-         marketIdsUpdated.add(id);
 
-         log.info("Updated market price for id={}, oldPrice={}, newPrice={}, normalDist={}", id, oldPrice, newMarketPrice, (newMarketPrice - oldPrice));
+//         log.info("Updated market price for id={}, oldPrice={}, newPrice={}, normalDist={}", id, oldPrice, newMarketPrice, (newMarketPrice - oldPrice));
```

Comment:
Could delete that?

Reply:
Yes I should, I was wondering whether we might want it in the backend for debugging but it is easy to re-write either way